backend/render_router.py

```
@@ -158,12 +190,19 @@ async def render_metrics(
             raise _unavailable(exc) from exc
         return {"service_id": service_id, "metrics": payload}
 
-    @router.get("/ops/status")
-    async def render_ops_status(user: dict = Depends(get_current_user)) -> dict:
-        """Counters for the autonomous Render monitoring loop."""
+    @router.get("/ops/status", response_model=RenderOpsStatus)
+    async def render_ops_status(
+        user: dict = Depends(get_current_user),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)